framework/py/flwr/server/superlink/fleet/grpc_rere/fleet_servicer.py

```diff
@@ -288,8 +288,12 @@ def PushObject(
                 state=self.state_factory.state(),
                 store=self.objectstore_factory.store(),
             )
-        except InvalidRunStatusException as e:
-            abort_grpc_context(e.message, context)
+            # Record traffic
+            bytes_recv = len(request.object_content)
+            state = self.state_factory.state()
```

Comment:
Since we are also calling `.state()` a few lines above, is it better if we do it just once? wdyt @panh99 ?